Modules/@babylonjs/react-native/shared/BabylonNative.h

```diff
@@ -8,8 +8,10 @@ namespace Babylon
 
     void Initialize(facebook::jsi::Runtime& jsiRuntime, Dispatcher jsDispatcher, bool autoRender = true);
     void Deinitialize();
-    void UpdateView(void* windowPtr, size_t width, size_t height, void* windowTypePtr = nullptr);
+    void UpdateView(void* windowPtr, size_t width, size_t height, void* windowTypePtr = nullptr, void* contextPtr = nullptr, void* backBufferPtr = nullptr);
     void RenderView();
+    void EnableView();
+    void DisableView();
```

Comment:
Are these actually used? Maybe I just missed it, not sure!

Reply:
This code is not used.